GuidelinesForNewComponents.md

```diff
@@ -0,0 +1,28 @@
+# Guidelines for Contributing New Components to Open Data Hub
+
+This document describes the guidelines to submit new ideas such as new features and components to Open Data Hub. This specifically pertains to new components or large features that require an architecture review or a re-design of existing components. This documents does not apply to small or incremental features, for those please open an issue with a "feature" label in the [Open Data Hub project](https://issues.redhat.com/projects/ODH/issues/ODH-380?filter=allopenissues)
+
+## Basic Requirements for Component
+Proposed components must have the following basic requirements:
+1. The component is an open source component with a valid open source license.
+2. The component should be aligned with Open Data Hub goals, architecture and roadmap.
+3. All build and source files for the component must be publicly available. 
+4. For component images and builds it is recommended to use [Red Hat Universal Base Images](https://catalog.redhat.com/software/containers/search?q=ubi&p=1)(UBI) based images.
```

Comment:
Add a space between the url and `(UBI)` - it is rendered like this at the moment:  
> Red Hat Universal Base Images(UBI)